go-controller/hybrid-overlay/pkg/controller/master.go

```diff
@@ -289,13 +290,87 @@ func (m *MasterController) handleOverlayPort(node *kapi.Node, annotator kube.Ann
 		}
 	}
 
+	// create a reroute on the ovn_cluster_router
+	err = createLogicalRouterPolicyForNode(node, subnets)
+	if err != nil {
+		return fmt.Errorf("failed to create logical router policy: %v", err)
+	}
+
+	return nil
+}
+
+func createLogicalRouterPolicyForNode(node *kapi.Node, subnets []*net.IPNet) error {
+	// get all the windows subnets as the dest for lr-policy match
+	var dest string
+	var nextHop string
+	var src string
+	for _, hybridSubnet := range config.HybridOverlay.ClusterSubnets {
+		if !utilnet.IsIPv6CIDR(hybridSubnet.CIDR) {
+			if len(dest) > 0 {
+				dest = dest + " || "
+			}
+			dest = dest + fmt.Sprintf("ip4.dst == %s", hybridSubnet.CIDR.String())
+		}
+	}
+	//the src is the nodes subnet and the next hop is the third address (x.x.x.3) in the nodes subnet
+	for _, subnet := range subnets {
+		if !utilnet.IsIPv6CIDR(subnet) {
+			src = fmt.Sprintf("ip4.src == %s", subnet.String())
+			ip := util.GetNodeHybridOverlayIfAddr(subnet)
+			nextHop = ip.IP.String()
+			break
+		}
+	}
+	if len(nextHop) == 0 {
+		return fmt.Errorf("could not find an ipv4 Node subnet for node %s", node.Name)
+	}
+
+	_, stderr, err := util.RunOVNNbctl(
+		"--id=@lr-policy",
+		"create",
+		"logical_router_policy",
+		"priority=1006",
```

Comment:
make the priority a constant similar to what we are doing for other policy routes

Reply:
@trozet PTAL at what I did, I took from a different PR moving the ovn_cluster_router constants to the config directory so that any package can import the constants and use them